packages/core/src/model.js

```diff
@@ -1759,6 +1763,18 @@ ${associationOwner._getAssociationDebugList()}`);
     }).save(options);
   }
 
+  static _searchInclude(values) {
+    const association = [];
+    const associationKeys = Object.keys(this.associations);
+    for (const value in values) {
+      if (associationKeys.includes(value)) {
+        association.push({ association: value });
```

Comment:
Also I think you can just push `value` and that the `{ association }` wrapper is redundant